spec/services/feature_service_spec.rb

```diff
@@ -0,0 +1,56 @@
+require "rails_helper"
+
+describe FeatureService do
+  let(:feature_enabled) { true }
+
+  around(:each) do |example|
+    old_value = Settings.features.rspec_testing
```

Comment:
Can we just stub this with `allow(Settings.features).to receive(:rspec_testing)` to save setting and unsetting.

Reply:
In `FeatureService` we access the setting like so:

```ruby
      Settings.features[feature_name]
```

so I was imagining we need to be a bit more sophisticated than just stubbing it out.

Reply:
```
  before do
    allow(Settings.features).to receive(:[]).with(:rspec_testing).and_return(feature_enabled)
  end
```

Reply:
My reasoning was that I'd rather it wasn't tied to whether we are doing `Setting.features.rspec_testing` or `Setting.features[:rspec_testing]`.

If that's not sufficiently convincing I can change it to use a stub.

Reply:
Ok. Fair enough 👍